Makefile

```diff
@@ -86,7 +86,7 @@ golangci_lint=$(shell which golangci-lint)
 $(OUTDIR)/tools/golangci-lint:
 	mkdir -p $(OUTDIR)/tools
 ifeq ($(golangci_lint),)
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(OUTDIR)/tools" v1.63.1
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(OUTDIR)/tools" v1.64.5
```

Comment:
Security: Don't love running shell scrips off the web. We can use our own, something like:
```bash
#!/bin/bash

version=1.64.5
os=$(go env GOOS)
arch=$(go env GOARCH)
url=https://github.com/golangci/golangci-lint/releases/download/v${version}/golangci-lint-${version}-${os}-${arch}.tar.gz

curl -sL ${url} |
	tar \
		-C ${OUTDIR}/tools \
		--strip-components=1 \
		-xzf - golangci-lint-${version}-${os}-${arch}/golangci-lint
```

Reply:
I understand your hesitation, but this is the official way to install this tool for generic CI, see here:

https://golangci-lint.run/welcome/install/#other-ci

I updated the link in line 83